packages/material-ui/src/Checkbox/Checkbox.test.js

```diff
@@ -14,16 +17,13 @@ describe('<Checkbox />', () => {
     mount = createMount({ strict: true });
   });
 
-  after(() => {
-    mount.cleanUp();
-  });
-
   describeConformance(<Checkbox checked />, () => ({
     classes,
     inheritComponent: IconButton,
     mount,
     refInstanceof: window.HTMLSpanElement,
     skip: ['componentProp'],
+    after: () => mount.cleanUp(),
```

Comment:
Yeah that is a bit obscure if you're unfamiliar with the testing setup. Glad that you figured this out for yourself :+1: